### ISSUE TITLE
v3.2.6 - Unable to customize language_client

### ISSUE THREAD

**Issue by bkuhl** [Mermade/widdershins#127]
I've configured my app to use [http2 objects over curl](https://github.com/Kong/httpsnippet/blob/master/src/targets/php/http2.js), but it still generates curl.

My config:

```
{
    "httpsnippet": true,
    "resolve": true,
    "language_tabs": [
        {"shell":"Shell"},
        {"php":"PHP"},
        {"java":"Java"},
        {"javascript":"Javascript"}
    ],
    "language_clients": [
        {"php":"http2"}
    ]
}
```

> I know config is being utilized because the language_tabs are correct.

Using 2x `--verbose` my output is:

```
+ widdershins@3.2.6
updated 1 package and moved 1 package in 2.379s
starting deref Customer Data API
finished deref
Compiling all doT templates...
Loaded def authentication.def
Loaded def authentication_none.def
Loaded def callbacks.def
Loaded def debug.def
Loaded def discovery.def
Loaded def footer.def
Loaded def links.def
Loaded def parameters.def
Loaded def responses.def
Loaded def security.def
Compiling code_go.dot to function
Compiling code_http.dot to function
Compiling code_java.dot to function
Compiling code_javascript.dot to function
Compiling code_nodejs.dot to function
Compiling code_python.dot to function
Compiling code_ruby.dot to function
Compiling code_shell.dot to function
Compiling main.dot to function
Compiling operation.dot to function
Compiling translations.dot to function
```

Example docs (I've verified these are being regenerated)

![screen shot 2018-05-10 at 10 57 48 am](https://user-images.githubusercontent.com/524933/39876379-ffa5773c-5440-11e8-8690-f83b498389f5.png)

**Comment by MikeRalphson** [comment]
There was some confusion in the PR which added this functionality over the format of the `language_clients` option property. I've made some changes to make everything agree with the documentation. Just running CI tests.

**Comment by MikeRalphson** [comment]
Should be fixed in `v3.2.8`. If you're using node 6, you'll now need to specify the `--harmony` flag to node.js

**Comment by bkuhl** [comment]
Bingo!  That did it, thanks!
